l10n/gitahead_pt_BR.ts

```diff
@@ -3540,48 +3581,55 @@ Exemplos
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="509"/>
         <source>Delete</source>
-        <translation type="unfinished">Deletar</translation>
+        <translation>Deletar</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="529"/>
         <source>Push Tag to %1</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Empurre a Tag para %1</translatorcomment>
+        <translation>Empurre a Tag para %1</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="537"/>
         <source>New Local Branch</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Novo Branch Local...</translatorcomment>
+        <translation>Novo Branch Local</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="545"/>
         <source>Merge...</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Merge...</translatorcomment>
+        <translation>Merge...</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="557"/>
         <source>Rebase...</source>
-        <translation type="unfinished">Rebase...</translation>
+        <translatorcomment>Pega os commits do branch pai, sincroniza com os commits anterior desse, e joga os commits do branch atual no final da pilha.</translatorcomment>
+        <translation>Rebase...</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="569"/>
         <source>Squash...</source>
-        <translation type="unfinished">Merge --squash...</translation>
+        <translatorcomment>Mescla alocando as alterações do Branch especificado no topo, mesclando como um novo commit.</translatorcomment>
```

Comment:
Is this translatorcomment displayed anywhere? It seems like you're filling this in like a tooltip that's displayed to the user, but I believe it's more like documentation for future developers who might change/update the translation.

Reply:
Basically I inserted this comment to specify what each action does, currently the application does not have this explanatory feature that triggers the comment when you hover the mouse pointer over some action, I placed the observation just to facilitate the translation process, but in the future is a resource to think about...